src/sentry/tasks/post_process.py

```diff
@@ -310,7 +310,13 @@ def fetch_buffered_group_stats(group):
     queue="triggers-0",
 )
 def post_process_group(
-    is_new, is_regression, is_new_group_environment, cache_key, group_id=None, **kwargs
+    is_new,
+    is_regression,
+    is_new_group_environment,
+    cache_key,
+    group_id=None,
+    group_states: Optional["GroupStates"] = None,
```

Comment:
Nit: if you import `from  __future__ import annotations` at the top you don't need to use strings.